### ISSUE TITLE
Deprecation warnings from Doctrine namespace change

### ISSUE THREAD

**Issue by kdorsey** [Codeception/module-doctrine2#6]
Hello,

I ran `php vendor/bin/codecept run --steps` and received the following deprecation warnings:

```
DEPRECATION: The Doctrine\Common\Persistence\Mapping\Driver\AnnotationDriver class is deprecated since doctrine/persistence 1.3 and will be removed in 2.0. Use \Doctrine\Persistence\Mapping\Driver\AnnotationDriver instead. /var/www/html/vendor/doctrine/persistence/lib/Doctrine/Common/Persistence/Mapping/Driver/AnnotationDriver.php:16
DEPRECATION: The Doctrine\Common\Persistence\ObjectManager class is deprecated since doctrine/persistence 1.3 and will be removed in 2.0. Use \Doctrine\Persistence\ObjectManager instead. /var/www/html/vendor/doctrine/persistence/lib/Doctrine/Common/Persistence/ObjectManager.php:16
DEPRECATION: The Doctrine\Common\Persistence\Mapping\AbstractClassMetadataFactory class is deprecated since doctrine/persistence 1.3 and will be removed in 2.0. Use \Doctrine\Persistence\Mapping\AbstractClassMetadataFactory instead. /var/www/html/vendor/doctrine/persistence/lib/Doctrine/Common/Persistence/Mapping/AbstractClassMetadataFactory.php:16
DEPRECATION: The Doctrine\Common\PropertyChangedListener class is deprecated since doctrine/persistence 1.3 and will be removed in 2.0. Use \Doctrine\Persistence\PropertyChangedListener instead. /var/www/html/vendor/doctrine/persistence/lib/Doctrine/Common/PropertyChangedListener.php:16
DEPRECATION: The Doctrine\Common\Persistence\Mapping\RuntimeReflectionService class is deprecated since doctrine/persistence 1.3 and will be removed in 2.0. Use \Doctrine\Persistence\Mapping\RuntimeReflectionService instead. /var/www/html/vendor/doctrine/persistence/lib/Doctrine/Common/Persistence/Mapping/RuntimeReflectionService.php:16
```

The test suite and framework do run to completion, but I get these warnings each time I run it, I thought it would be good to share these as a heads up.

These PRs appear to be related:
https://github.com/doctrine/orm/pull/7953
https://github.com/doctrine/persistence/pull/71

**Comment by Naktibalda** [comment]
Doctrine2 module doesn't use any of these classes, please double check yourself at https://github.com/Codeception/module-doctrine2/blob/master/src/Codeception/Module/Doctrine2.php

**Comment by beberlei** [comment]
We released a 1.3.4 of persistence and rolled back the deprecation messages coming from `doctrine/orm`, and will only emit them when 1.4 is released. The idea is that all external code depends on `doctrine/persistence` version `^1.3` and then the ORM and/or `mongodb-odm` will raise to 1.4 in the near future when they are compatible. This way we will defer the deprecation messages until we ourselves are compatible.
